docs/tutorials/connecting-to-the-internet-via-usb.md

```diff
@@ -3,126 +3,97 @@ title: Connecting to the Internet via USB
     If you are using OS X version 10.11, this won't work. Apple broke the USB/CDC
     gadget driver and has not fixed it yet (as of version 10.11.2). Please
     [file a bug with Apple](https://bugreport.apple.com/) and politely ask them
     to fix it.
-    {:class="alert alert-warning"}
+    {: class="alert alert-warning"}
 
-    1.  On the EV3, first verify that the CDC driver is enabled. In brickman,
-        open the *Wireless and Networks* menu and select *USB*. Make sure *CDC* is
-        selected and active.
+    <span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
+    These instructions were written using OS X version 10.9, but should work for
+    other versions as well.
+    {: class="alert alert-info"}
 
-        {% include screenshot.html source="/images/brickman/usb-cdc-active.png" %}
+    1.  Use the USB cable provided with the EV3 to connect the EV3 to your Mac.
+        On your Mac, open up *System Preferences* and select *Network*.
 
-    2.  On your Mac, open up *System Preferences* and select *Network*.
+        {% include screenshot.html source="/images/osx/10.9/SystemPreferencesNetwork.png" %}
 
-        {% include screenshot.html source="/images/OSXRNDIS/SystemPreferencesNetwork.png" %}
-
-    3.  Once you get the *Network* dialog, click on the `+` icon in the lower left
+    2.  Once you get the *Network* dialog, click on the `+` icon in the lower left
         area to add a new network device. You'll need to select the CDC Composite
         Gadget in the drop down box. The name will be similar to what is shown
         below. I have renamed the service "ev3dev" so it's easier to keep track of
         later. Click *Create* when you are done.
 
-        {% include screenshot.html source="/images/OSXRNDIS/AddNewDevice.png" %}
+        {% include screenshot.html source="/images/osx/10.9/AddNewDevice.png" %}
 
-    4.  Click *Apply* to save your changes. After a short time, the ev3dev entry
+    3.  Click *Apply* to save your changes. After a short time, the ev3dev entry
         (or whatever you named it) should show connected and have a Self-Assigned
         IP address
 
-        {% include screenshot.html source="/images/OSXRNDIS/CDC-Connected.png" %}
-
-    5. If you don't need to access the Internet from your EV3 over this
-        connection, manually configure the IP address to `192.168.2.1`. Note:
-        you won't be able to update packages without an Internet connection.
-
-        TODO: Need detailed description with screenshots.
-
-        Then skip to step 7.
+        {% include screenshot.html source="/images/osx/10.9/CDC-Connected.png" %}
 
-        If you do want to be able to connect to the Internet from the EV3, continue
-        with the next step.
-
-    6.  To share our Internet connection with the EV3. Go back to *System
+    4.  To share our Internet connection with the EV3. Go back to *System
```

Comment:
That... Doesn't make any sense :wink: I think that was originally a different sentence that was restructured.